docs/cloud/azure/architecture.md

```diff
@@ -1 +1,939 @@
+
+:fontawesome-regular-map:
+[Disaster-recovery planning for kdb+ tick systems](../..//wp/disaster-recovery/index.md)
+<br>
+:fontawesome-regular-map:
+[Data recovery for kdb+ tick](../../wp/data-recovery.md)
+
+## Network
+
+Network bandwidth needs to be considered if the tickerplant
+components are not located on the same VM. The network bandwidth between
+Azure VMs depends on the type of the VMs. For example, a VM of type
+`Standard_D8as_v4` has an expected network bandwidth 3.125 Gbps and a
+larger instance `Standard_D32as_v4` can sustain 12.5 Gbps. For a given
+update frequency you can calculate the required bandwidth by employing the
+[`-22!`](../../basics/internal.md#-22x-uncompressed-length)
+internal function that returns the length of the IPC byte representation
+of its argument. The tickerplant copes with large amounts of data if
+batch updates are sent. Make sure that the network is not your
+bottleneck in processing the updates.
+
+### Azure Load Balancer
+
+An Azure Load Balancer is a type of load balancing service by Azure. It
+is used for ultra-high performance, TLS offloading at scale, centralized
+certificate deployment, support for UDP, and static IP addresses for
+your application. Operating at the connection level, network load
+balancers are capable of handling millions of requests per second
+securely while maintaining ultra-low latencies.
+
+Load balancers can distribute load among applications that offer the
+same service. Kdb+ is single threaded by default. With a negative -p
+parameter you can set multithreaded input mode in which requests are
+processed in parallel. This however, is not recommended for gateways
+(due to socket-usage limitation) and for kdb+ servers that process data
+from disk, like HDBs. A better approach is to use a pool of HDB
+processes. Distributing the queries can either be done by the gateway
+via async calls or by a load balancer. If the gateways are sending sync
+queries to the HDB load balancer, then a gateway load balancer is
+recommended to avoid query contention in the gateway. Furthermore, there
+are other tickerplant components that enjoy the benefit of load
+balancers to better handle simultaneous requests.
+
+Adding a load balancer on top of an historical database (HDB) pool is
+quite simple, it only needs three steps. 
+
+1.   Create a Network Load Balancer with protocol TCP. Set the name, Availability Zone, Target Group name and Security group. The security group needs an inbound rule to the HDB port. 
+2.   Create a launch template. A key part here is the User Data window where you can type a startup-script. It mounts the volume that contains the HDB data and the q interpreter, sets environment variables (e.g. `QHOME`) and starts the HDB. The HDB accepts incoming TCP connections from the Load Balancer so you need to set up an inbound Firewall rule via a Security Group. You can also leverage an Image (AMI) that you already created from an existing EC2. 
```

Comment:
Let's change: 
~~You can also leverage an Image (AMI) that you already created from an existing EC2.~~
You can also leverage a Custom Image that you already created from an existing Azure VM.